src/axom/quest/readers/C2CReader.cpp

```diff
@@ -978,15 +978,19 @@ void C2CReader::getLinearMeshNonUniform(
   FILE* fthresh = fopen("threshold.curve", "wt");
   fprintf(fthresh, "# threshold\n");
 #endif
-  int contourCount = -1;
+
+  int contourCount {-1};
+
+  // clang complains about contourCount (-Wunused-but-set-variable); since we want keep it, let's mark it
+  AXOM_UNUSED_VAR(contourCount);
 
   // Iterate over the contours and linearize each of them.
   std::vector<Segment> S;
   S.reserve(INITIAL_GUESS_NPTS);
   for(const auto& nurbs : m_nurbsData)
   {
     NURBSInterpolator interpolator(nurbs, m_vertexWeldThreshold);
-    contourCount++;
+    ++contourCount;
```

Comment:
Totally a style call, but why is this worth a change?

Reply:
I think it was just a reflexive change while I was hunting down the warning about `contourCount` in some configuration/platform.

Reply:
Okay.